raiden_contracts/utils/signature.py

```diff
@@ -8,15 +8,23 @@
 
 
 def sign(privkey: str, msg: bytes, v: int = 0) -> bytes:
-    assert isinstance(msg, bytes)
```

Comment:
Just wondering, should we use privkey: str or privkey: bytes to get rid of potential encoding errors here?

Reply:
Good point.  I added an issue to track this https://github.com/raiden-network/raiden-contracts/issues/1059 .